### PR TITLE
Fixes #23957 - Use proper Notications API for bulk products

### PR DESCRIPTION
**Pull request by jturel**
Three messages to be tested all from the products list with at least one selected product:

- Sync Selected
- Remove
- Advanced Sync -> select one of the options

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js [hunk 1]
```diff
@@ -27,23 +27,10 @@ angular.module('Bastion.tasks').config(['$stateProvider', function ($stateProvid
         abstract: true,
         templateUrl: 'tasks/views/tasks.html'
     })
-    .state('tasks.index', {
-        url: 'katello_tasks',
-        permission: 'view_tasks',
-        templateUrl: 'tasks/views/tasks-index.html'
-    })
-    .state('tasks.details', {
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js]: I don't think these are used, searched with something like `git grep \'tasks.index\' engines/`

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js [hunk 2]
```diff
@@ -27,23 +27,10 @@ angular.module('Bastion.tasks').config(['$stateProvider', function ($stateProvid
...
-        url: 'katello_tasks/:taskId',
-        permission: 'view_tasks',
-        collapsed: true,
-        controller: 'TaskDetailsController',
-        templateUrl: 'tasks/views/task-details-standalone.html'
-    })
     .state('task', {
-        url: 'katello_tasks/single/:taskId',
+        url: 'foreman_tasks/tasks/:taskId',
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js]: Fixes the url in the notification

**Comment by jturel** [comment]
Nice catch ... fixed and did some small cleanup

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js (hunk 1 above)
Comment (akofink) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js]: Try `git grep \"tasks.details\" engines/` to find [it](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/views/tasks-index.html#L1).

Comment (jturel) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks.module.js]: Thanks - I restored the tasks.details state. I didn't investigate but I think it's possible the view referencing it is not actually used. Maybe some tech debt, but not going to clean it all up here.

**Pull request merged**